stdlib/test/builtin/test_string.mojo

```diff
@@ -209,6 +213,12 @@ fn test_atol() raises:
     except e:
         assert_equal(str(e), "String is not convertible to integer.")
 
+    try:
```

Comment:
**Suggestion** You can use [assert_raises](https://github.com/modularml/mojo/blob/e98f496c3a0d3565f1d6453412d5530894f4bc1e/stdlib/src/testing/testing.mojo#L272) from the `testing` module.  If you'd rather defer this and potentially refactor the other uses in this file to use `assert_raises`, that's totally fine.

Reply:
Oh, I didn't know assert_raises was already implemented, I'll probably change that I'm these days